docs/group-user-management.rst

```diff
@@ -0,0 +1,253 @@
+Administrate Groups and Users
+=============================
+
+Description:
+------------
+
+This chapter will show how to manage groups and users using the graphical interface in OMERO.web and the command-line interface. Most of following tasks below can only be done by users with some
+administrator privileges. We will show:
+
+- How to manage groups, creating and editing a new/existing group
+- How to manage users, creating and editing a new/existing user
+
+Resources:
+----------
+
+-  Documentation:
+
+   -  https://docs.openmicroscopy.org/latest/omero/sysadmins/server-permissions.html
+
+   -  https://docs.openmicroscopy.org/latest/omero/sysadmins/restricted-admins.html
+
+   -  https://docs.openmicroscopy.org/omero/latest/sysadmins/cli/usergroup.html
+
+   -  https://docs.openmicroscopy.org/omero/5.6.1/sysadmins/server-ldap.html
+
+   -  https://docs.openmicroscopy.org/omero/5.6.1/developers/Server/Ldap.html?highlight=ldap
+
+
+-  Script for Command Line User/Group management
+
+   - https://github.com/ome/training-scripts/blob/master/maintenance/scripts/create_groups_users.sh
+
+-  File defining the User/Group setup used by the script
+
+   - https://github.com/ome/training-scripts/blob/master/maintenance/scripts/create_groups_users_setup
+
+Setup:
+------
+
+No setup needed for OMERO.web administration panel (see Web Interface chapter below) except working OMERO.web.
+
+**Command Line interface installation**
+
+The installation instructions can be
+found at \ https://docs.openmicroscopy.org/latest/omero/users/cli/installation.html\ .
+
+
+Step-by-step:
+-------------
+
+Administrate using the Web Interface
+------------------------------------
+
+#. In your web browser, go to the server address provided.
+
+#. Log in using the username and password provided.
+
+#. In the top toolbar, click the Admin button |image0|\ . Note that the Admin button is only available for users with certain privileges (administrators and administrators with restricted privileges). If you are a user or a group owner, navigate to the section "Web Interface: Users change their own settings" below.
+
+*Web Interface: Managing Groups*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. Click on the Groups tab. You can search for groups if desired
+
+#. To create a new Group, click on the Add new Group button. Note that the Name and Permissions fields are mandatory.
+
+#. Click Save.
```

Comment:
After "Click Save". Add: "The new group will be shown in the list of Groups".
This helps explain the next bullet point better.

Reply:
Fixed in https://github.com/ome/omero-guide-introduction/pull/1/commits/a184195e748d1a077500817e4ed7e3738d4e39b0